packages/connect-examples/expo-example/webpack.config.js

```
@@ -46,6 +46,46 @@ module.exports = async function (env, argv) {
+        '[hash]',
+        `${commitSha}-[hash]`
+      );
+    }
+
+    // 修改 CSS 文件名
+    const miniCssExtractPlugin = config.plugins.find(
+      plugin => plugin.constructor.name === 'MiniCssExtractPlugin'
+    );
+    if (miniCssExtractPlugin && miniCssExtractPlugin.options) {
+      if (miniCssExtractPlugin.options.filename) {
+        miniCssExtractPlugin.options.filename = miniCssExtractPlugin.options.filename.replace(
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)